README.md

```diff
@@ -17,44 +17,60 @@ NOTE: We do not guarantee the quality of responses produced by this sample copil
 
 ## Step 1: Set up your development environment
 
-#### Use a pre-built development environment
-To get started quickly, you can use a pre-built development environment. **Click the button below** to open the repo in GitHub Codespaces, and then continue the readme!
-
+### Step 1a: Use a cloud development environment
+#### Explore sample with Codespaces
+- To get started quickly with this sample, you can use a pre-built Codespaces development environment. **Click the button below** to open this repo in GitHub Codespaces, and then continue the readme!
 [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/Azure/aistudio-copilot-sample?quickstart=1)
 
-Once you've opened in Codespaces you can proceed to the next step.
+- Once you've launched Codespaces you can proceed to step 2.
+
+#### Start developing in an Azure AI curated VS Code development environment
+- If you intend to develop your own code following this sample, we recommend you use the Azure AI curated VS Code development environment. It comes preconfigured with the Azure AI SDK and CLI that you will use to run this sample.
+
+- **If you are viewing this README from within that VS Code container already, you can proceed directly to step 2!**
 
-#### Alternatively, set up your local development environment
+- You can get started with this cloud environment from the Azure AI Studio by following these steps: [Work with Azure AI projects in VS Code](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/develop-in-vscode)
 
-First, clone the code sample locally:
+### Step 1b: Alternatively, set up your local development environment
+
+1. First, clone the code sample locally:
 ```
 git clone https://github.com/azure/aistudio-copilot-sample
 cd aistudio-copilot-sample
 ```
 
-Create a new Python virtual environment where we can safely install the SDK packages:
+2. Next, create a new Python virtual environment where we can safely install the SDK packages:
+
+:warning: If you are using Python 3.12, there is a package dependency we are working to resolve. Check your python version with `py -3 --version`, and if the version is 3.12, use the commented 3.12 alternative for creating the virtual environment
+
  * On MacOS and Linux run:
    ```
    python3 -m venv .venv
+   # if 3.12: python3 -m venv .venv
```

Comment:
It seems like we provide the same command for this line and the last line above?

Reply:
Yes, this is incomplete - I am asking what it should be!! please share with me the correct command for MacOS and Linux

Reply:
I tried, for 3.12, it's `python 3.11 -m venv .venv`, and then I can use `source .venv/bin/activate` to use version 3.11

Reply:
with a space between python and 3.11 @YusakuNo1